src/lib/Libnet/net_server.c

```diff
@@ -89,7 +89,8 @@
 static conn_t **svr_conn;    /* list of pointers to connections indexed by the socket fd. List is dynamically allocated */
 #define CONNS_ARRAY_INCREMENT	100 /* Increases this many more connection pointers when dynamically allocating memory for svr_conn */
 static int conns_array_size = 0;  /* Size of the svr_conn list, initialized to 0 */
-
+pbs_sched	*psched;
+pbs_list_head	svr_allscheds;
```

Comment:
Remove these variables and define wherever required. As this code is part of Libnet library we need to keep generic variables and functions only.

Reply:
done